neurom/ezy/tests/test_neuron.py

```diff
@@ -73,6 +73,10 @@ def setUp(self):
         self.filename = os.path.join(SWC_PATH, 'Neuron.swc')
         self.neuron = ezy.Neuron(self.filename)
 
+    def test_name(self):
+        print self.neuron.filename
+        nt.assert_true(self.neuron.filename == os.path.join(SWC_PATH, 'Neuron'))
```

Comment:
We remove the extension and keep the "real name". This comes from the neuron.id (check io.utils.load_neuron code, where this comes from.)